docs/data/data-grid/server-side-data/ServerSideErrorHandling.tsx

```diff
@@ -110,7 +110,7 @@ export default function ServerSideErrorHandling() {
         <DataGridPro
           {...props}
           unstable_dataSource={dataSource}
-          unstable_onDataSourceError={(e) => setError(e.message)}
+          unstable_onDataSourceError={(err) => setError(err.message)}
```

Comment:
This is 99% stylistic, but `err` is only slightly better than `e`, for two reasons:
1. it's still an unnecessary abbreviation.
2. it seems to be chosen here because it shadows an existing `error`.

but for 2. it's only technically not shadowing the existing variable. Just by looking at the name "err" vs. "error" I can't make a distinction between the two. It's still shadowing the variable in spirit IMO.

The way I usually tend to solve this naming problem is by declaring nested errors/events with a more precise name, for example:

```suggestion
          unstable_onDataSourceError={(dataSourceError) => setError(dataSourceError.message)}
```